docs/tutorial/fdfitting.rst

```diff
@@ -0,0 +1,314 @@
+
+
+We can have a quick look at what this model looks like and which parameters are in there::
+
+    >>> pylake.inverted_odijk("DNA")
+
+    Model equation:
+
+    f(d) = argmin[f](norm(DNA_Lc \left(1 - \frac12\sqrt{\frac{kT}{f DNA_Lp}} + \frac{f}{DNA_St}\right) - d))
+
+    Parameter defaults:
+
+    Name      Value  Unit      Fitted      Lower bound    Upper bound
+    ------  -------  --------  --------  -------------  -------------
+    DNA_Lp    40     [nm]      True                  0            inf
+    DNA_Lc    16     [micron]  True                  0            inf
+    DNA_St  1500     [pN]      True                  0            inf
+    kT         4.11  [pN*nm]   False                 0              8
+
+This seems to be what we want. Force as a function of distance. Now let's say we have acquired data,
+but we notice that the template matching wasn't completely optimal. And that we had some force drift,
+while forgetting to reset the force back to zero. In this case, we can incorporate offsets in our
+model. We can introduce an offset in the independent parameter, by calling `subtract_independent_offset`
+on our model::
+
+    odijk_with_offset = pylake.inverted_odijk("DNA").subtract_independent_offset("d_offset")
+
+If we also expect an offset in the dependent parameter, we can simply add an offset model to our
+model::
+
+    odijk_with_offsets = pylake.inverted_odijk("DNA").subtract_independent_offset("d_offset") + pylake.force_offset("f")
+
+From the above example, you can see how easy it is to composite models. Sometimes, models become more 
+complicated. For instance, we may have two worm like chain models that we wish to add, and then invert.
+For the Odijk model, this can be done as follows::
+
+    two_odijk = (pylake.odijk("DNA") + pylake.odijk("protein") + pylake.force_offset("f")).invert()
+
+
+Note how we added three models and then inverted the composition of those models. The parentheses 
+are important here, since otherwise we would have only inverted the offset model. Note that models
+inverted via `invert()` will typically be slower than the pre-inverted counterparts. This is because
+the inversion is done numerically, rather than analytically. For more complex examples on how this
+inversion may be used, please see the examples.
+
+For a full list of models that are available, please refer to the documentation by invoking
+`help(pylake.fitting.models)`.
+
+
+Loading data
+------------
+
+Next up, is loading some data. Let's assume we have two data sets. One was acquired in the presence
+of a ligand, and another was measured without a ligand. We expect this ligand to only affect the 
+contour length of our DNA. Loading the first dataset is simple. Note that we explicitly define
+which dataset is force and which is distance via keyed arguments (this is intended to prevent
+mistakes in argument order)::
+
+    data1 = odijk_with_offsets.load_data(f=force1, d=distance1, name="Control")
```

Comment:
What can the user do with the returned `data1` object? It's not used in the example. I would either remove it or show an example. If it's advanced usage, you can remove the variable here and put it in a separate section on this page.

Reply:
It is used in the example down the road. It's used to pick parameters from the `Fit` as `odijk_fit[data1]`. Do you think we should try and put those closer together? Then I would probably duplicate the earlier code down the road.